modules/logging-loki-log-access.adoc

```diff
@@ -74,16 +74,36 @@ subjects:
 // tag::CustomAdmin[]
 == Custom admin group access
 
-If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. Admin users have access to all application logs in all namespaces, if they also get assigned the `cluster-logging-application-view` role.
+// tag::LokiMode[]
+If you have a large deployment with a number of users who require broader permissions, you can create a custom group using the `adminGroup` field.  Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. 
+// end::LokiMode[]
+
+// tag::NetObservMode[]
+If you need to see cluster-wide logs without being an administrator, you can create a custom group using the `adminGroup` field.  Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR are considered admins. 
```

Comment:
Actually, there's another use-case that is relevant for this config, and I've seen customers mentioning it, it's when they already have some admin groups defined, but not named "cluster-admin", and they'd like to re-use it rather than creating a new one, so they can also do the same here.

So we could formulate it like:

```suggestion
If you need to see cluster-wide logs without necessarily being an administrator, or if you already have a group defined for cluster admins that you would like to use here, you can specify a custom group using the `adminGroup` field. Users who are members of any group specified in the `adminGroups` field of the `LokiStack` CR have the same read access to logs as admins. 
```

Reply:
Awesome, thanks for adding these details and the extra use case.